### ISSUE TITLE
moonbeam.network is not a phishing site

### ISSUE THREAD

**Issue by eshaben** [MetaMask/eth-phishing-detect#5086]
TLDR: Moonbeam is not a phishing site. Can you please remove the incorrect flagging?

## Description
Brave browser is issuing a phishing detection warning for a Polkadot parachain site, https://moonbeam.network/. The site is legitimate and contains no solicitations of user information. It's a static website.

Note: there is a subdomain for the documentation site that is hosted in mkdocs (https://docs.moonbeam.network/) that contains a "connect to MetaMask" button, but this is not malicious and simply allows users to connect their MetaMask to our TestNet: https://docs.moonbeam.network/getting-started/testnet/metamask/

## Steps to Reproduce
1. Navigate to https://moonbeam.network/
2. Receive warning from Brave

We received reports of this behavior as early as last week but were not able to reproduce until now. It seems to be related to the IP address of the user, since I am now traveling and accessed the website from a different location. Only experienced it on the browser so far.

## Actual result:
![image](https://user-images.githubusercontent.com/54632920/119664608-81fcb880-be01-11eb-94c5-e36f9fdcb730.png)
 
## Expected result:
![image](https://user-images.githubusercontent.com/54632920/119664668-92ad2e80-be01-11eb-9d55-ad43e8d21cb9.png)
 
## Reproduces how often:
Intermittent issue: does not happen for every user every time.

**Comment by jacobcantele** [comment]
I've confirmed this is a legitimate issue with the Moonbeam team.

**Comment by Gudahtt** [comment]
Thanks for the report! But it looks like we're not blocking moonbeam.network, so there's not much that can be done in this repo. This looks like a problem with Brave specifically - I'd suggest asking them about this maybe?

I'll close this for now, since the problem doesn't lie with this repo.
